resources/local/config.toml

```diff
@@ -89,6 +89,9 @@ systemd_support = false
 # The actual bound address will be reported via a log line if logging is enabled.
 address = '0.0.0.0:7777'
 
+# The max rate of requests (per second) before they are limited.
```

Comment:
I think the comment should also state if this is a per-client limit or not, and also what happens when the limit is hit.  (I'm guessing the client sending connecting is temporarily blocked rather than being dropped?)

We need to duplicate that info in all 6 places I guess.